src/masonite/providers/RouteProvider.py

```diff
@@ -39,7 +42,13 @@ def boot(self):
             exception = None
 
             try:
-                response.view(route.get_response(self.application))
+                data = route.get_response(self.application)
+                if isinstance(data, Response) or (
+                    isclass(data) and issubclass(data, ResponseFacade)
```

Comment:
This looks really messy too .. Maybe we check for a method or attribute on the object?